qingstor/sdk/client/upload_client.py

```diff
@@ -0,0 +1,82 @@
+from qingstor.sdk.service.qingstor import QingStor
+from file_chunk import FileChunk
+from ..constants import HTTP_CREATED,HTTP_OK,HTTP_BAD_REQUEST
+from ..error import PartTooSmallError,MaxPartsExceededError
+import logging
+import os
+class UploadClient:
+    '''file processing,including open file,move file pointer and read file
+
+    Parameter:
+        bucket: bucket refers to bucket object users creating in the QingCloud
+        part_size_MB(int):the part size users want to partition of the file in MB
+        part_size(int): the part size users want to partition of the file in byte
+        
+
+    Attributes:
+        file_descriptor(object): the source uploading file object
+        file_name: the key(usually using file name) of the uploading file
+        bucket: bucket refers to bucket object users creating in the QingCloud
+        part_size_MB(int): the part size users want to partition of the file in MB
+        part_size(int): the part size users want to partition of the file in byte
+        part_wait_list(int): part_wait_list(int): the list of part index which is waiting to upload
+        cur_read_part(str): the string of current part reading from the source file (from class FileChunk)        
+        part_amount(int): the total part number partitioned of the source file according to the part size (from class FileChunk)
+        this_upload_id: the unique upload id of the uploading file
+        file_chunk: the instance of the class FileChunk
+        part_index: the index of each part
+        part_uploaded_list: the list of part index which completes uploading
+        
+    
+    '''
+
+
+    def __init__(self,bucket,part_size_MB):
+    	if(part_size_MB<4):
+            raise PartTooSmallError()
+        else:
+            self.bucket=bucket
+            self.part_size=part_size_MB*1024*1024
+            self.logger=self.log_config()
+        
+    def upload_large_file(self,file_name,file_descriptor):
+        #initiate multipart upload, create an upload id
+        output=self.bucket.initiate_multipart_upload(file_name) 
+        if output.status_code==HTTP_OK:
+           self.logger.info("Multipart Upload Initialization Done!")   
+        else:
+            self.logger.error("Invalid Request!") 
```

Comment:
We should handle this error instead of just print a log.
And why we will have a `Invalid Request!` here ?